expan/core/experiment.py

```diff
@@ -141,3 +141,62 @@ def delta(self, method='fixed_horizon', **worker_args):
 
         result['kpis'] = kpis
         return result
+
+    @staticmethod
+    def _quant_apply(value, threshold):
+        if value < threshold:
+            return False
+        else:
+            return True
+
+    def _quantile_filtering(self, df, percentile):
+        flags = None
+        for column in df.columns:
+            if flags is None:
+                flags = df[column].apply(
+                    self._quant_apply,
```

Comment:
Are we planning to use different functions here? otherwise `lambda x, y: x >= y` would suffice and there is no need to declare the static method above